deps/media-playback/media-playback/decode.c

```diff
@@ -114,8 +114,15 @@ static uint16_t get_max_luminance(const AVStream *stream)
 {
 	uint32_t max_luminance = 0;
 
+// FFmpeg 6.1 puts the side data into AVCodecParameters.
+#if LIBAVCODEC_VERSION_MAJOR < 60 || LIBAVCODEC_VERSION_MINOR < 31
```

Comment:
This really should be something like `#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(60, 31, 100)` to avoid false-positives if there's a version `(61, 0, 0)` or similar.

Reply:
This has been fixed in the latest push. Thank you for the review.